gradlew

```diff
@@ -161,9 +161,4 @@ function splitJvmOpts() {
 eval splitJvmOpts $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS
 JVM_OPTS[${#JVM_OPTS[*]}]="-Dorg.gradle.appname=$APP_BASE_NAME"
 
-"$JAVACMD" "${JVM_OPTS[@]}" -classpath "$CLASSPATH" org.gradle.wrapper.GradleWrapperMain "$@"
-
-# stop the daemon on windows systems to close file locks left open on some files made by gradle
-if [ "$cygwin" = "true" -o "$msys" = "true" ]; then
-    "$JAVACMD" "${JVM_OPTS[@]}" -classpath "$CLASSPATH" org.gradle.wrapper.GradleWrapperMain --stop
-fi
+exec "$JAVACMD" "${JVM_OPTS[@]}" -classpath "$CLASSPATH" org.gradle.wrapper.GradleWrapperMain "$@"
```

Comment:
The "exec" is the important part so that we can run Gradle within the context of the wrapper, and avoid absorbing it's exit status.  

The other red part we might have added for one of our internal builds, to give some love to Windows users running the daemon, despite file locking problems at the time.  https://issues.gradle.org/browse/GRADLE-2415 may still be an issue?  

Should we keep it?